templates/terraform/constants/monitoring_slo.go.erb

```diff
@@ -4,4 +4,21 @@ func validateMonitoringSloGoal(v interface{}, k string) (warnings []string, erro
 		errors = append(errors, fmt.Errorf("goal %f must be > 0 and <= 0.999", goal))
 	}
 	return
-}
\ No newline at end of file
+}
+<% if compiler == "terraformobjectlibrary-codegen" -%>
+
+func expandMonitoringSloRollingPeriodDays(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
```

Comment:
Can we include this in a conversion-specific handwritten file?

Reply:
Sure, I will move this code to a handwritten file

Reply:
handwritten file created